src/Domain/Setting/test/SettingRepositoryStub.ts

```diff
@@ -23,4 +27,14 @@ export class SettingRepostioryStub implements SettingRepositoryInterface {
     
     return found
   }
+  /**
+   * Note: this doesn't really create or replace anything, just pretends it did.
+   */
+  async createOrReplace(dto: CreateOrReplaceSettingDto): Promise<CreateOrReplaceSettingStatus> {
```

Comment:
as mentioned I think this method is not required ... look at the following:

https://github.com/standardnotes/auth/blob/9f885ebf7603d687bfd6577a3af90bdd1e8b223c/src/Domain/Session/RevokedSessionRepositoryInterface.ts#L5

https://github.com/standardnotes/auth/blob/9f885ebf7603d687bfd6577a3af90bdd1e8b223c/src/Infra/MySQL/MySQLRevokedSessionRepository.ts#L8

Reply:
Same issue: https://github.com/standardnotes/auth/pull/12#discussion_r607061719